data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				set "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report begins with an economic analysis of the cost of raids at Wah Ki, which it places at a staggering 3.5 billion credits per day. This is 1.5 billion impact on trade; 1.5 billion for military fleet repairs, replacements, ammunition, and infrastructure to support it; and an estimated 500 million in humanitarian aid to the Unfettered, to prevent the problem from worsening. It compares that to "full Hai unification" for which the cost is 7 billion credits per day: 3 billion in humanitarian aid to the Unfettered, 5 billion in military aid, but 1 billion per day recouped through improved trade.`
+			`	The remaining 80% of the report is redacted in its entirety, save for a few section headings:`
+			`	-	Multi-Decadal Timeline Expected before Unfettered Progress`
+			`	-	Choot'k: Insane Risk-Taker or Brilliant Strategist?`
+			`	-	Reports from Hai Agents on Unfettered Raiding Vessels`
+			`	-	Unsustainable Rate of Losses Spying on Unfettered Leadership`
+			`	-	Theories on the Sentient Evolution of Nocturnal Raptors`
+			`	Musing on the titles, you recall that "raptors" are birds of prey. Why would an intelligence report on the Unfettered discuss nocturnal birds of prey?`
+				goto reading
+
+			label acorns
+			action
+				log `A Hai intelligence report said Unfettered planets have been ruined to the point where they cannot only produce enough food for bare sustenance. Birds, predators, and large species were almost wiped out, destroying the ecosystem. It mentioned three species of oak that could grow, but none are ideal. As time goes on Unfettered will increasingly rely on external food sources.`
+				set "Hai Reveal [A11] Sound the Alarm: acorns book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report describes the catastrophic environmental collapse of the Unfettered worlds through over-farming, lack of pollution controls, and over-mining. Nearly all large species, including predators and nearly all birds, have been wiped out, ruining the ecosystems. Presently, the Unfettered can produce slightly more food needed for bare sustenance on their own, and are reliant on donations or raids for the rest.`
+			`	There is a detailed study on alternative crops, mentioning that many worlds have evolved a plant like an oak tree, and some will grow readily on the ruined Unfettered world of Warfeed. The swamp white oak from Earth is known to have healing properties, but is insufficient as a food source. The Earth's willow oak has powerful aphrodisiac and stimulant effects. There is a "Mile-High Blue Oak" species on Featherweight with powerful healing effects and is a rapid food supply, but takes two hundred years to begin producing acorns. This means the Unfettered need another source of food.`
```

Comment:
I dont mind this but if they knew about this food crisis why did they not attempt a long term solution?
I think saying that could be more interesting than acorns but we could have both
Say such an action was attempted but failed because of smth (likely the unfettered not managing to maintain it in the long term)

Reply:
This is not supposed to be a comprehensive list of possibilities, but if you have anything specific to suggest, let me know and I'll write it up.

Reply:
I have updated it based on your suggestions. How does it look now?

Reply:
the phrasing is a bit odd and you havent spoken much of it but its not really needed I guess
and idk about "beyond the ones the unfet have already accepted"